translations/monero-core_sv.ts

```diff
@@ -590,7 +590,7 @@
     <message>
         <location filename="../pages/Mining.qml" line="145"/>
         <source>Error starting mining</source>
-        <translation>Fel vid start av brytning</translation>
+        <translation>Ett fel uppstod vid start av brytning</translation>
     </message>
```

Comment:
Mining har tidigare översatts till utvinning, då det är drn tidigare etablerade termen.

Reply:
Håller med om att det är lite knivigt att välja ord här. Jag såg i höstas att det pågick översättning men hade inte tid att hjälpa till (blanda mig i...). Var ni flera som diskuterade, och i så fall var? Det finns en kanal i slack (#monero-se) utan någon aktivitet. Och jag hittar inte den kanalen i Mattermost.

Mitt subjektiva resonemang är att "utvinna" associerar till något som inte kräver så mycket arbete. T ex kan man utvinna salt ur havsvatten genom att pumpa upp det i en damm och sedan bara vänta på att solen och värmen gör sitt, och när vattnet har dunstat ligger saltet där.

Men i krypto handlar det ju om att utföra arbete och t.om skapa ett bevis för detta arbete som läggs in i blocket. Och då tycker jag att "bryta" mer associerar till hårt arbete. Och det är väl det som är ursprunget till metaforen "mining" på engelska?

Google tycks ge fler träffar på "bryta bitcoin" än "utvinna bitcoin" (inget man nödvändigvis behöver ta hänsyn till). 

Vad tycker du?

Reply:
Just utvinning togs på den generella svenska listan, mitt förslag mynta fick stå tillbaka då utvinning redan var etablerat i flertalet bitcoin projekt.

Reply:
tp-sv@listor.tp-sv.se är listan jag talar om. Är du inte på den, så rekommenderas det

Reply:
Bra tips, ska gå med där. 

Hittade er konversation på listan och ditt förslag på 'mynta' tycker jag var överraskande. Det är ju inte "minting" (eng.) utan "mining" (eng.) det handlar om. I krypto framställer man inte pengar utan utför arbete som om man har tur leder till att man får en ersättning i form av valuta. Hela metaforen på engelska handlar ju på gruvdrift, Bitcoin är digtalt guld, media skriver om bitcoin-gruvor osv. Så varför tycker du att det inte skulle passa på svenska? 

Det enda jag ser om 'utvinning' är kommentaren

"Jag har av någon anledning fått för mig att det heter att utvinna BitCoins."

Är det allting bygger på, att någon har fått för sig något? (hoppas att jag inte låter otrevlig för det är inte meningen. Men lite resonemang och argument hade varit intressant)